### ISSUE TITLE
Automod rules do not synchronise in some situations

### ISSUE THREAD

**Issue by fsvreddit** [fsvreddit/automod-sync#9]
Due to inconsistent line ending handling

**Comment by d-reddi** [comment]
Hi there -
I've noticed the sync is still not happening.  I've tried it different ways, if I edit the main rule (i.e., the regex) slightly, and then save.. try going back to the main sub running sync, it says no rules need to sync.  It will only sync if I delete the rule entirely, then re-add the #include, then save again.  Then of course, it overwrites the comment information I've changed etc. 

Steps: 

1. Add an include line to the automod wiki #include <sub> <rule name>
2. Save --- Observe rule has been synced
3. Go back into automod wiki, in the primary regex, delete some code to force a re-sync.
4. Save
5. Go into sub, press sync automoderator
6. Receive message that no rules need to be synced.  (Even though you've edited the main regex, and it should sync back to the original).
